docker-compose.yml

```diff
@@ -22,11 +23,15 @@ services:
       - postgres
       - rabbitmq
       - redis
+      - splash
     env_file:
       - .env
+    environment:
+      - SCRAPY_SETTINGS_MODULE=gazette.settings
+      - PYTHONPATH=/mnt/code/data_collection
     volumes:
-      - ${DATA_FOLDER}:/mnt/data
-      - ./processing:/mnt/code
+      - /c/Users/Daniela/Documents/chico/diario-oficial/data:/mnt/data
```

Comment:
Developer environment data! Don't change it!

Reply:
Yeah, that won't be kept in the end. This is still WIP, though.

Reply:
#58 to allow me to use the weird boot2docker absolute paths...